.github/workflows/reusable-update-spec.yml

```diff
@@ -21,9 +23,11 @@ jobs:
         id: cpr
         uses: peter-evans/create-pull-request@v4
         with:
-          commit-message: Automatic ${{ inputs.type }} spec update from upstream changes
+          commit-message: |
+            Automatic ${{ inputs.type }} spec update from upstream changes
           delete-branch: true
-          title: 'Automatic ${{ inputs.type }} spec update'
+          title: |
```

Comment:
Why has this been changed? Title is always a single line, right? Is this a personal preference or does this have an advantage?

Reply:
You are free in a block!
In quotes you have to fight YAML.